pkg/api/addons_test.go

```diff
@@ -7608,15 +7589,600 @@ func TestSetAddonsConfig(t *testing.T) {
 					},
 					Containers: []KubernetesContainerSpec{
 						{
-							Name:  common.KubeDNSAddonName,
+							Name:  "kubedns",
```

Comment:
Again, more errors slipping through because we weren't actually validating kube-dns in this UT.